Sources/SnapshotTestingExtensions/Extensions/SwiftUI/View.swift

```diff
@@ -1,6 +1,6 @@
 // Copyright © 2023 Lautsprecher Teufel GmbH. All rights reserved.
 
-#if DEBUG && canImport(SwiftUI)
+#if DEBUG && canImport(SwiftUI) && canImport(UIKit)
```

Comment:
I opened a PR to update [swift-snapshot-testing](https://github.com/pointfreeco/swift-snapshot-testing) dependency, and it fixes build error. Therefore, we can remove that check.